test/unit/url.test.js

```diff
@@ -1700,6 +1700,12 @@ describe('Url', function () {
                 expect(url.toString()).to.eql('https://postman-echo.com////////get/');
             });
 
+            it('getPath->unresolved should handle empty path properly', function () {
+                var url = new Url('{{host}}/');
+
+                expect(url.getPath(false)).to.be.equal('/');
```

Comment:
Shouldn't we test the `getPath(true)` since we made modifications for the true case?